src/java.base/share/classes/java/lang/ClassValue.java

```diff
@@ -53,48 +78,59 @@ protected ClassValue() {
     }
 
     /**
-     * Computes the given class's derived value for this {@code ClassValue}.
+     * Computes the value to associate to the given {@code Class}.
      * <p>
-     * This method will be invoked within the first thread that accesses
-     * the value with the {@link #get get} method.
+     * This method is invoked when an initial read-only access by {@link #get
+     * get} failed to find this association.
      * <p>
-     * Normally, this method is invoked at most once per class,
-     * but it may be invoked again if there has been a call to
-     * {@link #remove remove}.
+     * If this method throws an exception, the initiating {@code get} call will
+     * not attempt to associate a value, and may terminate either by returning
+     * an observed associated value, if it exists, or by propagating that
+     * exception.
```

Comment:
Does this mean that exceptions may be swallowed?

Reply:
Yes, if a value is discovered to be associated when an exception from computeValue was trying to be thrown.